dart/2.Dart_basics/Excercises/2_exercise.md

```diff
@@ -0,0 +1,11 @@
+# Exercise 2
+
+Declare two integer variables called x and y and initialize them with any values. Then, use asserts to make sure that x is greater than y, and print the message "Assertion passed" if the assertion is true, or "Assertion failed" if the assertion is false.
```

Comment:
do you think would be correct to use  `x` and `y` to reference them? Meaning code?